app/views/homescreen/blocks/_news.html.erb

```diff
@@ -1,15 +1,14 @@
 <%= render 'homescreen/blocks/header', title: t(:label_news_latest) %>
-
 <% unless @news.empty? %>
   <ul class="widget-box--arrow-links">
   <% @news.each do |news| %>
     <li class="-widget-box--arrow-multiline">
       <div>
-        <%= avatar(news.author, {class: 'avatar-mini'}) %>
+        <%= avatar(news.author, {class: 'avatar author-avatar'}) %>
         <%= link_to_project(news.project) + ': ' %>
         <%= link_to h(news.title), news_path(news) %>
-        <br/>
-        <span class="additional-information"><%= authoring news.created_at, news.author %></span>
+        <div class="author"> <span><%= authoring news.created_at, news.author %></span> </div>
```

Comment:
Please try to avoid opening multiple HTML tags in one line. It is very hard to read and to understand where one element starts and where it ends. Instead use a new line for each tag and use indentations for hierarchy. This does not only apply for this line, but for all your changes.
So for example: 
```
<div class="author">
  <span><%= authoring news.created_at, news.author %></span>
</div>
```